go/interpreter/lfvm/interpreter_test.go

```diff
@@ -232,141 +220,76 @@ func TestInterpreter_step_DetectsUpperStackLimitViolation(t *testing.T) {
 	}
 }
 
-// TODO: this needs to be eliminated
```

Comment:
NOTE: although out of scope, it was quite difficult to refactor this unit test. The code has been deleted here, but a migration branch exists already:
https://github.com/Fantom-foundation/Tosca/pull/753